deployer/deployer.yaml

```diff
@@ -67,13 +66,35 @@ items:
       - watch
       - delete
       - update
+  -
+    apiVersion: v1
+    kind: RoleBinding
+    metadata:
+      name: logging-deployer-edit-role
+    roleRef:
+      kind: ClusterRole
+      name: edit
+    subjects:
```

Comment:
yes! unfortunately we can't do the same thing yet for ClusterRoleBindings
or SCC membership. But we can at least save the step for roles local to the
project.

On Fri, Jun 17, 2016 at 10:12 AM, Eric Wolinetz <notifications@github.com>
wrote:

> In deployer/deployer.yaml
> <https://github.com/openshift/origin-aggregated-logging/pull/158#discussion_r67516129>
> :
>
> > @@ -67,13 +66,35 @@ items:
> >        - watch
> >        - delete
> >        - update
> > +  -
> > +    apiVersion: v1
> > +    kind: RoleBinding
> > +    metadata:
> > +      name: logging-deployer-edit-role
> > +    roleRef:
> > +      kind: ClusterRole
> > +      name: edit
> > +    subjects:
>
> so this lets us avoid needing to manually grant the deployer
> serviceaccount these roles?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin-aggregated-logging/pull/158/files/2646176167971eb8d5b2c80f21270ec4ebebfa4a#r67516129>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AABz-opEbcuD_hPMWzeHBVUatvW0X-hJks5qMqthgaJpZM4Ir7Nu>
> .
>